src/playermat/PlaymatApi.ttslua

```diff
@@ -3,26 +3,70 @@ do
   local internal = { }
 
   local MAT_IDS = {
-    White = "8b081b",
+    White  = "8b081b",
     Orange = "bd0ff4",
-    Green = "383d8b",
-    Red = "0840d5"
+    Green  = "383d8b",
+    Red    = "0840d5"
   }
 
   local CLUE_COUNTER_GUIDS = {
-    White = "37be78",
+    White  = "37be78",
     Orange = "1769ed",
-    Green = "032300",
-    Red = "d86b7c"
+    Green  = "032300",
+    Red    = "d86b7c"
   }
 
   local CLUE_CLICKER_GUIDS = {
-    White = "db85d6",
+    White  = "db85d6",
     Orange = "3f22e5",
-    Green = "891403",
-    Red = "4111de"
+    Green  = "891403",
+    Red    = "4111de"
   }
 
+  -- Returns the by color requested playermat as object
+  ---@param matColor String Color of the playermat to return
+  PlaymatApi.getMatbyColor = function(matColor)
```

Comment:
Is anything using this?  A major point of the API is that it's an interface to get to other objects, returning the actual one seems counter to that.

Reply:
Not yet - I added it for completeness.

Reply:
I think we should remove this.  Not only isn't it being used I think it's something we don't want to be used.  So if we put it in now it's either dead code until we do remove it, or gets used and is something else we have to clean up.

Reply:
Done